app/views/layouts/admin.html.haml

```diff
@@ -28,6 +28,11 @@
         %p.govuk-phase-banner__content
           %strong.govuk-tag.govuk-phase-banner__content__tag
             beta
+          %span.govuk-phase-banner__text
+            This is a new service – your
+            %a.govuk-link{:href => "mailto:report-mi@crowncommercial.gov.uk"} feedback
```

Comment:
We're hard wiring the email address again here, this is what we wanted to avoid with out 'global' support address, see what you think of this:

```
          %span.govuk-phase-banner__text
            This is a new service – your
            = mail_to support_email_address, 'feedback', class: 'govuk-link'
            will help us to improve it.
```